### ISSUE TITLE
Geomap: Missing focus indicator on buttons on Firefox

### ISSUE THREAD

**Issue by szet0018** [wet-boew/wet-boew#8823]
On Firefox 72 our team at AAFC noticed that the buttons on the map do not show any focus indicator. They show fine on Internet Explorer 11.

We are wondering if there is another keyboard navigation besides TAB that should be used on the map.

**Comment by thekodester** [comment]
Firefox does not seem to apply focus when tabbing, but instead applies hover.

Tried with:
https://getbootstrap.com/docs/3.3/components/#btn-groups
https://wet-boew.github.io/wet-boew-styleguide/design/buttons-en.html

I know of a few accessibility issue with Firefox, you can try checking out their bug page for similar reports (https://bugzilla.mozilla.org/home).

###### I am just a volunteer on this GitHub project.
